src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -204,46 +204,48 @@ WaveFunctionComponent* SlaterDetBuilder::buildComponent(xmlNodePtr cur)
       if (multislaterdet_0 || multislaterdetfast_0)
         APP_ABORT("multideterminant is already instantiated.");
 
-      std::string spo_alpha_name;
-      std::string spo_beta_name;
+      int nGroups = targetPtcl.groups();
```

Comment:
I found it a good habit to put a const in this case when a local short name variable is used for convenience. const protects unintended modification.

Reply:
done